src/common/components/modal/Modal.tsx

```diff
@@ -9,23 +9,25 @@ type Props = {
   onClose: () => void;
 };
 
-const _Modal: ForwardRefRenderFunction<HTMLElement, PropsWithChildren<Props>> =
-  ({ children, isOpen, onClose }, externalRef) => {
-    if (!isOpen) {
-      return null;
-    }
-    return (
-      <Portal>
-        <Wrapper ref={externalRef} coords={{ y: window.scrollY }}>
-          <ExitButton onClick={onClose}>
-            <Times />
-          </ExitButton>
-          <Content>{children}</Content>
-        </Wrapper>
-        <BgBlur coords={{ y: window.scrollY }} />
-      </Portal>
-    );
-  };
+const _Modal: ForwardRefRenderFunction<
```

Comment:
Question unrelated to the code, are those just formatting changes? Github shows the diff but the code looks identical to me

Reply:
Probably from prettier.